src/aquarium-optimized/dawn/SeaweedModelDawn.h

```diff
@@ -46,8 +46,12 @@ class SeaweedModelDawn : public SeaweedModel {
     float specularFactor;
   } mLightFactorUniforms;
 
+  struct Seaweed {
+    float time;
+    float padding[3];
+  };
   struct SeaweedPer {
-    float time[20];
+    Seaweed seaweed[20];
```

Comment:
Why do we need a struct with just one field here?

Reply:
I don't think there's a good reason. Just try to be correct, and be consistent with the D3D12 backend ATM.

Reply:
OK, let's clean it up in a later patch.

Reply:
Yes, this should be dynamically allocated like fishes(DBO or instance draw).

Reply:
But I think it won't impact on performance because threre are only 3 or 4 seaweeds in the scene.